### PR TITLE
ENG-1094: orgs for users

### PR DESCRIPTION
**Pull request by stormcloud266**
- get/create org in service
- create missing orgs on sign up
- update seed user id

### CODE REVIEW

File: api/src/functions/auth/auth.ts [hunk 1]
```diff
@@ -74,12 +75,36 @@ export const handler = async (event: APIGatewayEvent) => {
 
     const { data } = payload
 
+    const placeholder = nanoid()
+    const createOrgOptions = {
+      organization: {
+        create: {
+          name: placeholder,
```
Comment (illepic) [inline api/src/functions/auth/auth.ts:82]: Just verifying that these are never used as primary key. The DB is creating a cuid for these as primary key right?

File: scripts/seed.ts [hunk 2]
```diff
@@ -36,7 +36,7 @@ export default async () => {
     {
       id: 'clfbeh0x90000l6z61p1dix7g',
       email: 'tawnee.la@gmail.com',
-      authId: 'user_2Yx1tAJCMor3UNpJbcDiPk0ouY4',
+      authId: 'user_2amE2mKQHV1MQ0kn15We5aChwqu',
```
Comment (illepic) [inline scripts/seed.ts:39]: Add us all!

**Review by illepic — CHANGES_REQUESTED** [review]

File: api/src/functions/auth/auth.ts (hunk 1 above)
Comment (stormcloud266) [reply api/src/functions/auth/auth.ts:82]: Yes their ids are all created by the db, just name and slug are also required on orgs

File: scripts/seed.ts (hunk 2 above)
Comment (stormcloud266) [reply scripts/seed.ts:39]: All the others are correct, but since I was deleting myself to test the webhook, mine ended up changing

**Review by illepic — APPROVED** [review]

**Pull request merged**